src/rules/valid-describe.ts

```diff
@@ -51,14 +64,14 @@ export default {
           }
 
           const [name] = node.arguments;
-          const [, callbackFunction] = node.arguments;
+          let [, callbackFunction] = node.arguments;
           if (!isString(name)) {
             context.report({
               messageId: 'firstArgumentMustBeName',
               loc: paramsLocation(node.arguments),
             });
           }
-          if (callbackFunction === undefined) {
+          if (!callbackFunction) {
```

Comment:
Could we do the `true` branch first, then `else`?

Reply:
sure